solidity/dashboard/src/css/liquidity-page.less

```diff
@@ -1,13 +1,23 @@
-.liquidity__card {
-  position:relative;
+.liquidity-banner {
+  margin: 0.8rem 3.2rem;
+  min-width: 25rem;
+  max-width: 100%;
+  display: flex;
+
+  svg.liquidity-banner__keep-logo {
+    height: 2rem;
+    width: 2rem;
+    margin-right: 1rem;
+  }
 
-  > .liquidity-card__santa-hat {
-    position: absolute;
-    z-index: 1;
-    right: -2.5rem;
-    top: -2rem;
+  .liquidity-banner__content {
+    .liquidity-banner__info {
+      margin: 0.3rem 0;
+    }
   }
+}
 
+.liquidity__card {
```

Comment:
I believe we should add `align-self: flex-start`, so the card height will depend on the content. Figma link [here](https://www.figma.com/file/Su61MwHLrQn3ld24q85pMx/Token-Dashboard-dApp?node-id=8130%3A223).

Current view:
![obraz](https://user-images.githubusercontent.com/57687279/103746771-e8736c80-5001-11eb-8564-822f26f789bd.png)

Reply:
69a44eb